src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java

```diff
@@ -173,6 +177,10 @@ public class KualiDocumentActionBase extends KualiAction {
     private BusinessObjectMetaDataService businessObjectMetaDataService;
     private AdHocRoutingService adHocRoutingService;
     private WorkflowDocumentService workflowDocumentService;
+    
+    //CU customization: Added blacklist attachment processing
+    private static final String ERROR_UPLOADFILE_EXTENSION = "error.uploadFile.extension";
```

Comment:
Should this constant be moved to CUKFSKeyConstants?

Reply:
Moved constant.